x-pack/plugins/lens/public/xy_visualization_plugin/xy_config_panel.tsx

```diff
@@ -0,0 +1,248 @@
+              data-test-subj="lnsXY_xDimensionPanel"
+              render={datasource.renderDimensionPanel}
+              nativeProps={{
+                columnId: state.x.accessor,
+                filterOperations: () => true,
+              }}
+            />
+          </EuiFormRow>
+
+          <EuiFormRow>
+            <EuiSwitch
+              label="Show gridlines"
+              data-test-subj="lnsXY_xShowGridlines"
+              checked={state.x.showGridlines}
+              onChange={() =>
+                setState({ ...state, x: { ...state.x, showGridlines: !state.x.showGridlines } })
+              }
+            />
+          </EuiFormRow>
+        </>
+      </EuiFormRow>
+
+      <EuiFormRow label="Y Axis">
+        <>
+          <EuiFormRow label="Title">
+            <EuiFieldText
+              placeholder="Title"
+              data-test-subj="lnsXY_yTitle"
+              value={state.y.title}
+              onChange={e => setState({ ...state, y: { ...state.y, title: e.target.value } })}
+              aria-label="Title"
+            />
+          </EuiFormRow>
+
+          <EuiFormRow label="Value">
+            <>
+              {state.y.accessors.map(accessor => (
+                <div key={accessor}>
+                  <NativeRenderer
+                    data-test-subj={`lnsXY_yDimensionPanel_${accessor}`}
+                    render={datasource.renderDimensionPanel}
+                    nativeProps={{
+                      columnId: accessor,
+                      filterOperations: (op: Operation) =>
+                        !op.isBucketed && op.dataType === 'number',
+                    }}
+                  />
+                  <EuiButtonIcon
+                    size="s"
+                    color="warning"
+                    data-test-subj={`lnsXY_yDimensionPanel_remove_${accessor}`}
+                    iconType="trash"
+                    onClick={() => {
+                      datasource.removeColumnInTableSpec(accessor);
+                      setState({
+                        ...state,
+                        y: {
+                          ...state.y,
+                          accessors: state.y.accessors.filter(col => col !== accessor),
+                        },
+                      });
+                    }}
+                    aria-label="Remove"
+                  />
+                </div>
+              ))}
+              <EuiButton
+                data-test-subj="lnsXY_yDimensionPanel_add"
+                onClick={() =>
+                  setState({
+                    ...state,
+                    y: {
+                      ...state.y,
+                      accessors: [...state.y.accessors, datasource.generateColumnId()],
+                    },
+                  })
+                }
+                iconType="plusInCircle"
+              />
+            </>
+          </EuiFormRow>
+
+          <EuiFormRow>
+            <EuiSwitch
+              label="Show gridlines"
```

Comment:
Are we taking care of i18n later? I added it in some places but not sure what makes more sense in our case.

Reply:
Ah. Interesting. I thought we were going to take care of it later. But I could be wrong, and it's probably easier to go ahead and do it as we go.

Reply:
I discussed this with @timroes and based on experience with i18n-ifying code after the fact it is more efficient to do this up front.